.github/workflows/test.yml

```
@@ -0,0 +1,38 @@
+name: Run tests
+
+on:
+  workflow_run:
+    workflows: [build]
+    types: [completed]
+
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        python: ["3.8", "3.9", "3.10", "3.11"]
+    runs-on: ${{ matrix.os }}
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+          path: dist/
+      - name: Install wheel and test dependencies
+        run: |
+          python -m pip install --upgrade pip
+          find ./dist/*.whl | xargs python -m pip install
```

```suggestion
          python -m pip install .
```

Install from source (which will also go through building a wheel, but just on the fly without having to download the pre-built wheel)